components/org.wso2.carbon.identity.api.user.approval/org.wso2.carbon.identity.rest.api.user.approval.v1/src/gen/java/org/wso2/carbon/identity/rest/api/user/approval/v1/dto/PropertyDTO.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
```suggestion
 *  http://www.apache.org/licenses/LICENSE-2.0
```
Let's generate the license. There are some indentations issues.

Reply:
Resolved in commit 0d3d04a371158df7d3e177f0ebd6f2138f38959e